SPECS/qemu-dp.spec

```diff
@@ -98,10 +98,16 @@ Patch87: make-blockdev-snapshot-use-same-node-name.patch
 Patch88: use_existing_io_context.patch
 Patch89: use_libaio_by_default.patch
 Patch90: Use_the_legacy_grant_copy_ioctl
+
+# XCP-ng patches
+Patch1000: qemu-dp-2.10.2-add-rbd-support.XCP-ng.patch
+
 BuildRequires: libaio-devel glib2-devel
 BuildRequires: libjpeg-devel libpng-devel pixman-devel libdrm-devel
 BuildRequires: xen-dom0-devel xen-libs-devel libusbx-devel
 BuildRequires: libseccomp-devel
+BuildRequires: librados-devel
```

Comment:
I think this dependency is probably not necessary because I think librdb-devel already requires librados-devel